changes.go

```diff
@@ -223,17 +223,40 @@ type TopicInput struct {
 	Topic string `json:"topic,omitempty"`
 }
 
+type Account struct {
+	AccountId int      `json:"_account_id"`
+	Name      string   `json:"name"`
+	Email     string   `json:"email"`
+	Username  string   `json:"username"`
+	Tags      []string `json:"tags,omitempty"`
+}
+
+type SubmitRecordLabel struct {
+	Label     string   `json:"label"`
+	Status    string   `json:"status"`
+	AppliedBy *Account `json:"applied_by,omitempty"`
+}
+
 // SubmitRecord entity describes results from a submit_rule.
 type SubmitRecord struct {
```

Comment:
According to the docs, `SubmitRecordInfo` (https://gerrit-review.googlesource.com/Documentation/rest-api-changes.html#submit-record-info) and `SubmitRecord` (https://gerrit-review.googlesource.com/Documentation/rest-api-changes.html#submit-record) are different types. I would prefer if we create two different types as well.

Reply:
In a new commit I have modified